.github/workflows/add-pr-to-release-project.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: "add merged PRs to release party project"
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - closed
+
+jobs:
+  add_pr_to_release_project:
+    if: github.base_ref == 'main' && github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
+        with:
+          app_id: ${{ secrets.RELEASE_APP_ID }}
+          private_key: ${{ secrets.RELEASE_APP_PRIVATE_KEY }}
+      - name: Add PR to release project
+        env:
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+          PR_NUM: ${{ github.event.number }}
+          PR_REPO: ${{ github.event.repository.name }}
+          PR_TITLE: ${{ github.event.pull_request.title }}
+          PR_URL: ${{ github.event.pull_request.html_url }}
+        run: |
+          tools/scripts/add-pr-to-release-project.sh
```

Comment:
I don't think we want to directly run that embedded script from PRs (especially forks).

It might be nice to package that script up as a docker action, too, but perhaps that's overkill.  Just checking it out from the pull request target (or main) would probably be easier.

Reply:
I'd totally love help create and action for this if necessary.
We already have a repo for release PR's, so we may as well publish release tooling from it.